bootstrap/configure-aws-for-packer.tf

```diff
@@ -4,22 +4,37 @@ variable "region" {
   default     = "eu-west-1"
 }
 
-variable "bucket_name" {
+variable "vmie_bucket_name" {
   type        = "string"
   description = "S3 bucket to use for the VM import/export service"
 }
 
+variable "vagrant_bucket_name" {
+  type        = "string"
+  description = "S3 bucket to upload vagrant images to"
+}
+
 provider "aws" {
   region = "${var.region}"
 }
 
 resource "aws_s3_bucket" "vmie" {
-  bucket        = "${var.bucket_name}"
+  bucket        = "${var.vmie_bucket_name}"
   acl           = "private"
   force_destroy = true
 
   tags {
-    Name = "${var.bucket_name}"
+    Name = "${var.vmie_bucket_name}"
+  }
+}
+
+resource "aws_s3_bucket" "vagrant" {
+  bucket        = "${var.vagrant_bucket_name}"
+  acl           = "public-read"
```

Comment:
I'd be reasonably confident images that are open source are open source forever (much like travis ci and GitHub)